package.json

```diff
@@ -44,9 +44,8 @@
     "@types/selenium-webdriver": "^4.0.0",
     "@types/wavesurfer.js": "^3.3.1",
     "@typescript-eslint/eslint-plugin": "^4.0.0",
-    "@typescript-eslint/parser": "^4.13.0",
-    "acorn": "^8.0.0",
-    "ajv": "^7.0.0",
+    "@typescript-eslint/parser": "^4.0.0",
```

Comment:
The changes in this file should probably only be to the version part above (I expect this is an issue with the scripts,.

Reply:
I don't know how to handle this. These are all the values on the release branch, which it carried over here. I agree these changes shouldn't be there but I can't update the version number on the release branch for these libraries. These changes werent made by me or the script. They arise because the branch this commit arose from (the release branch) is behind develop.

Reply:
Hmm, okay, just merge it like this and then create a PR that will fix it?

Reply:
This slightly confuses me. Why didn't this branch originated from develop? Originating it from the release branch can result in us having to deal with conflicts from release-only cherrypicks. Since this is changing something in mainline, I'd expect us to only ever change the develop branch and never merge from release branch to develop.

Reply:
Hmmm, looks like I was confused myself. The branch did originate from develop but for some reason, it edited the release branch version of this file with the new oppia version (which I didn't expect). I'll take a look at why this issue arose, but I'll remove all these changes manually.